content/post/2023-08-25-friday-hacks-242.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Friday Hacks #242, August 25"
+date: 2023-08-25 12:25:15.5144
+author: Parth Gujar
+url: /2023/08/friday-hacks-242
+summary: On design philosophy and side projects with ChatGPT
+---
+
+**Date/Time:** Friday, August 25 at 7:00pm SGT<br />
+**Venue:** [**LT18** @ NUS Business](https://goo.gl/maps/mcp4cNysANSCYZCz9)<br />
+**Sign-up Link:** [Sign-up here](https://hckr.cc/links)<br />
+
+> **Food 🍕 will be served!**
+
+_If you are unable to come join us physically, you are welcome to join us [online](https://tr.ee/LQROP4Hwc8)!_
+
+<img src="/img/2023/fh/242.jpg" alt="Friday Hacks #242 Poster" /><br />
+
+## 1) Should engineers design?
+
+Designers design, and engineers build. This is the current state of things in the world. But is this the ideal state? Can engineers do more than just build?
+
+### Speaker Profile
+
+Chester, a product engineer at Mobbin (a UI/UX reference library for designers), challenges norms and shares his engineering philosophy. Also, he works on Design Spells, a newsletter showcasing design details 🪄
+
+## 2) Building a modular IoT transaction platform
```

Comment:
Title is wrong, can check the schedule/publicity poster for the right title

Reply:
Fixed